Classes/Views/CardCollectionViewCell.swift

```diff
@@ -94,23 +95,65 @@ class CardCollectionViewCell: UICollectionViewCell, UIGestureRecognizerDelegate
                 controlPoint: CGPoint(x: pixelSnapBounds.maxX, y: pixelSnapBounds.maxY)
             )
             borderPath.addLine(to: CGPoint(x: pixelSnapBounds.maxX, y: bounds.minY))
-
+            
+            fillPath = borderPath.copy() as! UIBezierPath
+            fillPath.close()
+        case .full:
+            
+            borderPath.move(to: CGPoint(x: pixelSnapBounds.minX,
+                                        y: bounds.maxY - cornerRadius))
```

Comment:
nit: can you follow the indent and newline style of the previous `case`s?

Reply:
Ya makes sense